src/styles/content.scss

```
@@ -0,0 +1,119 @@
+@import "mixins";
+
+.main {
+  display: flex;
+  flex-flow: wrap;
+  justify-content: space-between;
+  width: 944px;
+  margin: 50px auto 19px;
+}
+
+.card {
+  @include display-flex-column;
+  @include content-font();
+  box-sizing: border-box;
+  width: 200px;
+  height: 403px;
+  margin-bottom: 50px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product_photo {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  margin-top: 30px;
+  margin-bottom: 45px;
+}
+
+.card__product_photo img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__description {
+  box-sizing: border-box;
+  margin: 0 14px 5px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__content {
+  @include content-font();
```

```suggestion
  @include content-font;
```
думаю тут можна без дужок